sdk/dotnet/cmd/pulumi-language-dotnet/main.go

```diff
@@ -805,3 +814,96 @@ func (host *dotnetLanguageHost) GetProgramDependencies(
 		Dependencies: packages,
 	}, nil
 }
+
+func (host *dotnetLanguageHost) GenerateProject(
+	ctx context.Context, req *pulumirpc.GenerateProjectRequest) (*pulumirpc.GenerateProjectResponse, error) {
+	// Make a connection to the loader
+	conn, err := grpc.Dial(
+		req.LoaderAddress,
+		grpc.WithInsecure(),
+		rpcutil.GrpcChannelOptions(),
+	)
+	if err != nil {
+		return nil, errors.Wrapf(err, "language host could not make connection to loader")
+	}
+	loaderClient := codegenrpc.NewLoaderClient(conn)
```

Comment:
This is going to be the _worst_ part of this change, the "Loader" interface used all through codegen expects to be able to load entire schemas, so doing schema loading over grpc means we have to send the whole json schema over grpc and re-bind it client side.

Reply:
See: comment about using URIs. If we get an `https://` schema, plugin could cache that locally and reuse it between executions.